src/optimizer/join_order/relation_statistics_helper.cpp

```diff
@@ -0,0 +1,348 @@
+#include "duckdb/optimizer/join_order/relation_statistics_helper.hpp"
+
+#include "duckdb/catalog/catalog_entry/table_catalog_entry.hpp"
+#include "duckdb/function/table/table_scan.hpp"
+#include "duckdb/optimizer/join_order/join_order_optimizer.hpp"
+#include "duckdb/optimizer/join_order/relation_manager.hpp"
+#include "duckdb/planner/filter/conjunction_filter.hpp"
+#include "duckdb/planner/operator/logical_get.hpp"
+#include "duckdb/storage/data_table.hpp"
+
+namespace duckdb {
+
+struct DistinctCount;
```

Comment:
Same comment here as before, I don't think that this forward declaration is needed